client/asset/btc/btc.go

```diff
@@ -563,14 +563,49 @@ func init() {
 	asset.Register(BipID, &Driver{})
 }
 
+// nodeAccessor keeps a mutex for the underlying node. This is to block
+// access to the SPV wallet if it has gone into recovery and all the wallet
+// files have been deleted. Once the wallet goes into recovery, it cannot be
+// used again. Core will create a new ExchangeWallet and all references to the
+// current one will be removed, but in the meantime access to the old one
+// must be blocked.
+type nodeAccessor struct {
+	node       Wallet
+	mtx        sync.RWMutex
+	inRecovery bool
+}
+
+// getNode returns the node unless the node has gone into recovery.
+func (na *nodeAccessor) getNode() (Wallet, error) {
+	na.mtx.RLock()
```

Comment:
Because any node method callers will hang during recovery, I believe we'lll need to rethink this pattern.

It's also unfortunate that a non-spv / external wallet has to use the nodeAccessor when it has no purpose.

Will look closer and make an actionable suggestion.